### ISSUE TITLE
Cocoapods available

### ISSUE THREAD

**Issue by muhammadsaddamnur** [WalletConnect/WalletConnectSwiftV2#258]
for now, is this package available on cocoapods?

**Comment by MisterVants** [comment]
Cocoapods is not supported right now. We recommend using SPM.

**Comment by zkhalapyan** [comment]
Yeh, would rather not go back to the days of Cocoapods 🙂 SPM is awesome!

**Comment by muhammadsaddamnur** [comment]
i agree with that, SPM is awesome.
but to create flutter plugin, flutter not yet support SPM 😭 only Cocoapods 😭

**Comment by tinaszheng** [comment]
+1 for cocoapods support

**Comment by llbartekll** [comment]
@tinaszheng what is your reason for supporting cocoapods?

**Comment by alex-btl** [comment]
+1 for cocoapods. There's multiple frameworks that don't integrate with SPM (KMM, Flutter etc).

**Comment by smallerboy** [comment]
support cocoapods is wonderful

**Comment by flypaper0** [comment]
Cocoapods available in 1.0.5 release, check this out with: 
``` ruby
pod 'WalletConnectSwiftV2', :git => 'https://github.com/WalletConnect/WalletConnectSwiftV2.git', :tag => '1.0.5'
```
